### PR TITLE
[SD-4197] - Target subscribers for publish action in routing

### PR DESCRIPTION
**Pull request by akintolga**

### CODE REVIEW

**Comment by akintolga** [comment]
Depends on https://github.com/superdesk/superdesk-core/pull/350

**Comment by sjunaid** [comment]
:+1:

File: scripts/superdesk-ingest/module.js [hunk 1]
```diff
@@ -1077,6 +1078,27 @@
                     });
                 });
 
+                adminPublishSettingsService.fetchSubscribers().then(function(items) {
```
Comment (mdhaman) [inline scripts/superdesk-ingest/module.js]: look like this might fetch only top N subscribers not all.

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**